tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -33,48 +39,148 @@ func newMakeTxCmd(rootCfg *baseCfg, io commands.IO) *commands.Command {
 	)
 
 	cmd.AddSubCommands(
-		newAddPkgCmd(cfg, io),
-		newSendCmd(cfg, io),
-		newCallCmd(cfg, io),
-		newRunCmd(cfg, io),
+		NewMakeSendCmd(cfg, io),
 	)
 
 	return cmd
 }
 
-func (c *makeTxCfg) RegisterFlags(fs *flag.FlagSet) {
+func (c *MakeTxCfg) RegisterFlags(fs *flag.FlagSet) {
 	fs.Int64Var(
-		&c.gasWanted,
+		&c.GasWanted,
 		"gas-wanted",
 		0,
 		"gas requested for tx",
 	)
 
 	fs.StringVar(
-		&c.gasFee,
+		&c.GasFee,
 		"gas-fee",
 		"",
 		"gas payment fee",
 	)
 
 	fs.StringVar(
-		&c.memo,
+		&c.Memo,
 		"memo",
 		"",
 		"any descriptive text",
 	)
 
 	fs.BoolVar(
-		&c.broadcast,
+		&c.Broadcast,
 		"broadcast",
 		false,
 		"sign and broadcast",
 	)
 
 	fs.StringVar(
-		&c.chainID,
+		&c.ChainID,
 		"chainid",
 		"dev",
 		"chainid to sign for (only useful if --broadcast)",
 	)
 }
+
+func SignAndBroadcastHandler(
```

Comment:
This is somewhat related to the `ExecSignAndBroadcast` comment I made: I intentionally exported this (similar to every other `Handler`) because I believe this method is among the most useful. While working on a side project, I found myself needing to do exactly what this method does, which led to me duplicating almost every handler from this package into my own. The downside of exporting this is that we'll need to maintain it. However, since we're already exporting other handler methods, I don't think making this one available will have a significant impact.